stdlib/pagerduty/pagerduty.flux

```diff
@@ -10,6 +10,18 @@ builtin dedupKey
 option defaultURL = "https://events.pagerduty.com/v2/enqueue"
 
 
+// severity levels on status objects can be one of the following: ok,info,warn,crit,unknown
+// but pagerduty only accepts critical, error, warning or info.
+// formatSeverity turns a level from the status object into a pagerduty severity
+formatSeverity = (severity) => {
```

Comment:
Should this be called `severityFromLevel`  and then name the argument `level`? That would make it clear which way this function converts levels to severities. 

```
severityFromLevel(level) => ...
```